src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -39,8 +39,9 @@
  * are the list of ciphersuites to be accepted in an SSL/TLS/DTLS handshake,
  * the list of protocols to be allowed, the endpoint identification
  * algorithm during SSL/TLS/DTLS handshaking, the Server Name Indication (SNI),
- * the maximum network packet size, the algorithm constraints and whether
- * SSL/TLS/DTLS servers should request or require client authentication, etc.
+ * the maximum network packet size, the algorithm constraints, the signature
+ * schemes and whether SSL/TLS/DTLS servers should request or require client
+ * authentication, etc.
  * <p>
  * SSLParameters can be created via the constructors in this class.
  * Objects can also be obtained using the {@code getSSLParameters()}
```

Comment:
Since you introduce the terms "pre-populated" and "connection populated" in the new methods, I think it would be useful to describe them up front in the summary, ex:

`{@code SSLParameter} objects can be created via the constructors in this class, and can be described as pre-populated objects. {@code SSLParameter} objects can also be obtained using the ... <list all the methods>, and can be described as connection populated objects."

Reply:
Nice!  It is something I want to add for quite a while, but never get a chance.  The suggested description is really simple and great!

Updated!